bin/libs/component.ts

```diff
@@ -387,6 +387,20 @@ export function findAllLaunchComponents2(): string[] {
   });
 }
 
+function isClientAttls() {
+  const clientGlobalAttls = std.getenv("ZWE_zowe_network_client_tls_attls");
+  const serverGlobalAttls = std.getenv("ZWE_zowe_network_server_tls_attls");
+  let clientLocalAttls = std.getenv("ZWE_components_zaas_zowe_network_client_tls_attls");
+  let serverLocalAttls = std.getenv("ZWE_components_zaas_zowe_network_server_tls_attls");
+  let clientAttls = clientGlobalAttls || clientLocalAttls;
+  if ((clientGlobalAttls !== false) && (clientLocalAttls !== false) && (!clientAttls)) {
```

Comment:
The [`std.getenv`](https://github.com/JoeNemo/quickjs-portable/blob/291b874c516d10ceb103afe0bece7301a1cebef6/quickjs-libc.c#L655) is returning a string. This `if` is comparing string vs Boolean. You need to get Booleans:
```
const clientGlobalAttls = std.getenv("ZWE_zowe_network_client_tls_attls") == "true" ? true : false;
...
```
or comparing strings
```
if ((clientGlobalAttls !== "false") && (clientLocalAttls !== "false") && (!clientAttls)) {
```

I prefer the first version.